packages/suite/src/middlewares/wallet/coinjoinMiddleware.ts

```diff
@@ -278,5 +280,24 @@ export const coinjoinMiddleware =
             }
         }
 
+        if (action.type === COINJOIN.CLIENT_SESSION_PHASE) {
+            const { accountKeys } = action.payload;
+            const isAlredyInterrupted = api
+                .getState()
+                .wallet.coinjoin.accounts.find(({ key }) => key === accountKeys[0])
+                ?.session?.interrupted;
+
+            if (
+                (action.payload.phase === SessionPhase.CriticalError ||
+                    action.payload.phase === SessionPhase.AffiliateServerOffline) &&
```

Comment:
Not sure about that. Now it seems like there is no point in trying to reconnect at least.

Reply:
afilliate server can get online anytime makes no sense to pause because of AffiliateServerOffline